ui-tests/cypress/support/pageObjects/administration/fileBrowser/CreateFolderPage.js

```diff
@@ -6,18 +6,41 @@ import AppPage from '../../app/AppPage';
 
 import UploadFilesPage    from './UploadFilesPage';
 import CreateTextFilePage from './CreateTextFilePage';
+import FilesListPage from './FilesListPage';
 
 export default class CreateFolderPage extends Content {
 
-  breadCrumbs      = `${htmlElements.ol}.breadcrumb`;
-  operationButtons = `${htmlElements.div}.btn-group`;
+  breadCrumbs        = `${htmlElements.ol}.breadcrumb`;
+  operationButtons   = `${htmlElements.div}.btn-group`;
+
+  createFolderForm   = `${htmlElements.form}.FileBrowserCreateFolder`;
+  folderNameInput    = `${htmlElements.input}[name='path']#path`;
+  nameInputHelpBlock = `${htmlElements.span}.help-block`;
+  cancelButton       = `${htmlElements.a}.FileBrowserCreateFolderForm__btn-cancel`;
+  saveButton         = `${htmlElements.button}.FileBrowserCreateFolderForm__btn-submit`;
 
   getFileBrowserBreadCrumbs() {
     return this.getContents()
                .children(htmlElements.div).eq(2)
                .find(this.breadCrumbs);
   }
 
+  getBreadCrumbsElement(element) {
+    return this.getFileBrowserBreadCrumbs().children(htmlElements.li).eq(element);
+  }
+
+  clickBreadCrumbsRoot() {
+    this.getBreadCrumbsElement(0).click();
+    cy.wait('@openedRootFolder');
```

Comment:
where is this alias defined? what if I do not define it before calling this method? page object should always be stateless objects